src/t_set.c

```diff
@@ -1118,6 +1117,7 @@ void srandmemberWithCountCommand(client *c) {
         while (size > count) {
             dictEntry *de;
             de = dictGetFairRandomKey(d);
+            /* dictDelete(d, dictGetKey(de)); */
```

Comment:
?

Reply:
It is deleted in the lines below this using dictUnlink, sdsfree, dictFreeUnlinkedEntry. (This dict's type has no key destructor.) I tried things back and forth, but there's actually no change here. I'll delete this commented line.

Reply:
It looked weird from the diff I had open, but just looking at it now yeah it seems like a random line. I would vote to remove it yeah.